src/library-authoring/LibraryAuthoringPage.test.tsx

```diff
@@ -797,6 +922,12 @@ describe('<LibraryAuthoringPage />', () => {
     // No Types filter shown
     expect(screen.queryByRole('button', { name: /type/i })).not.toBeInTheDocument();
 
+    // Navigate to the units tab
+    fireEvent.click(await screen.findByRole('tab', { name: 'Units' }));
+    expect((await screen.findAllByText('Collection 1'))[0]).toBeInTheDocument();
```

Comment:
"Collection 1" is a collection, not a Unit.. could you ensure this test covers Unit type objects?

Reply:
I added a Unit here ([78f6a37](https://github.com/openedx/frontend-app-authoring/pull/1754/commits/78f6a37ec786b21367da8bc7f91d6892997ef1e1)), but we don't actually filter anything because we are just mocking the meilisearch result.